python/tk_framework_desktopserver/process_manager.py

```diff
@@ -195,6 +196,8 @@ def execute_toolkit_command(self, pipeline_config_path, command, args):
 
             return (out, err, return_code)
         except Exception, e:
+            # call_cmd is not including sentitive information in the error message, so this won't
```

Comment:
I put the comment in because this is the only occasion in this file where we are taking text coming from an outside source and trusting to add it to our own error string. All other places where we are raising exception is it clear that we've not adding sensitive information to the exception. I felt it would be good to underline the fact here since we have to be careful of what we log and what we don't.